packages/vue-app/template/components/nuxt-link.client.js

```diff
@@ -42,29 +33,36 @@ export default {
       default: '<%= router.linkPrefetchedClass %>'
     }<% } %>
   },
+  data () {
+    return {
+      __observed: false,
+      __unobserve: () => {}
```

Comment:
- Having `this.__unobserve` can imply it is observed
- We don't need to keep function as reactive data

Reply:
Moved to `$options` so it is not reactive anymore ☺️

Reply:
Or should I move it to the element itself? 🤔

Reply:
Unless there is a reason, i think instance should be fine :D